tests/Agent/IntegrationTests/IntegrationTests/WCF/WCFTestBase.cs

```diff
@@ -45,6 +45,9 @@ public enum ASPCompatibilityMode
         protected readonly ConsoleDynamicMethodFixtureFW _fixture;
 
         protected readonly WCFBindingType _bindingToTest;
+        protected string ExpectedTransportType => _bindingToTest == WCFBindingType.NetTcp
+            ? "NetTCP"
```

Comment:
I don't remember if we're testing NetMSMQ binding, but this PR might actually enable CAT and DT support for that binding type, because I think it's using a SOAP payload over a MSMQ transport.

Reply:
So, we are not testing these bindings.   But, I suspect they would work if we tested them.  The Transport Type would be OTHER, not MSMQ however.

Reply:
I don't think we should add tests for those binding types at this point. If something comes up then I think we can add some additional tests. As it stands WCF tests make up the vast majority of the tests in our integration test suite.

Reply:
I added an issue to track this.